docs/maintenance/pr-community-board.md

```diff
@@ -0,0 +1,41 @@
+# PR Community board workflow and guidelines
+
+**Intended audience**
+
+This guide is intended for contributors, coordinators and developers of InvenioRDM.
+
+**Scope**
+
+The guide covers the recommended way of coordinating community pull requests integration within the InvenioRDM. The goal of the project is to respond to the commmunity need of facilitating the review and integration processes of the **pull requests which are not the part of the main sprint**.
+
+## Board
+
+You'll find the latest board [here](https://github.com/orgs/inveniosoftware/projects/109/views/1).
```

Comment:
It would be helpful for the release manager to keep the PRs in the column until major release of invenio-app-rdm. It is very helpful to create release notes.